torchmetrics/functional/image/helper.py

```diff
@@ -83,6 +85,13 @@ def _single_dimension_pad(inputs: Tensor, dim: int, pad: int) -> Tensor:
 
 
 def _reflection_pad_3d(inputs: Tensor, pad_h: int, pad_w: int, pad_d: int) -> Tensor:
-    for dim, pad in enumerate([pad_h, pad_w, pad_d]):
-        inputs = _single_dimension_pad(inputs, dim + 2, pad)
+    (torch_v0, torch_v1) = torch.__version__.split(".")[:2]
+    if int(torch_v0) > 1 or int(torch_v1) >= 10:
```

Comment:
Please define rather helper variable
```python
_TORCH_GREATER_EQUAL_1_10: Optional[bool] = _compare_version("torch", operator.ge, "1.10.0")
```
in `torchmetrics/utilities/imports.py`

and then here check
```
if _TORCH_GREATER_EQUAL_1_10:
    ....
else:
    ....
```
to stay consistent with the rest of the repo :]

Otherwise good point to use torch implementation if possible, thanks for the fix :]

Reply:
Ok I'll do